packages/storybook/src/nimble/table/table.stories.ts

```diff
@@ -326,54 +334,101 @@ export const table: StoryObj<TableArgs> = {
         },
         checkValidity: {
             name: 'checkValidity()',
-            description:
-                'A function that returns `true` if the configuration of the table is valid and `false` if the configuration of the table is not valid.',
-            control: false
-        }
+            description: checkValidityDescription({ componentName: 'table' }),
+            control: false,
+            table: { category: apiCategory.methods },
+        },
+        setRecordHierarchyOptions: {
+            name: 'setRecordHierarchyOptions()',
+            description: 'Used to configure the hierarchy state of rows dynamically. To set static hierarchy, use `parent-id-field-name` instead. See the **Delay-loaded Hierarchy** section for more information.',
+            control: false,
+            table: { category: apiCategory.methods },
+        },
+        defaultSlot: {
+            name: 'default',
+            description: 'Configure table columns by adding them as child elements in the default slot. See **Table Column Configuration** for more information.',
+            control: false,
+            table: { category: apiCategory.slots },
+        },
+        actionMenuSlot: {
+            name: 'Action menus',
+            description: `To add an action menu to a column, add a \`${menuTag}\` element as a child of the table and set its slot attribute to a custom value. Then configure the column's \`action-menu-slot\` attribute to the same value. You can configure different action menus for multiple columns by giving them unique slot names.`,
+            control: false,
+            table: { category: apiCategory.slots },
+        },
+        actionMenuBeforetoggle: {
+            name: 'action-menu-beforetoggle',
+            description: 'Event emitted before an action menu opens or closes. This can be used to populate the menu before it is visible.',
+            control: false,
+            table: { category: apiCategory.events },
+        },
+        actionMenuToggle: {
+            name: 'action-menu-toggle',
+            description: 'Event emitted after an action menu opens or closes.',
+            control: false,
+            table: { category: apiCategory.events },
+        },
+        selectionChange: {
+            name: 'selection-change',
+            description: 'Event emitted when the user changes which rows are selected.',
+            control: false,
+            table: { category: apiCategory.events },
+        },
+        columnConfigurationChange: {
+            name: 'column-configuration-change',
+            description: 'Event emitted when the user changes a column\'s width or sort order.',
+            control: false,
+            table: { category: apiCategory.events },
+        },
+        rowExpandToggle: {
+            name: 'row-expand-toggle',
+            description: 'Event emitted when the user expands or collapses a row in a table with hierarchy.',
```

Comment:
Should we say that this is not emitted for group rows? (Or if it actually is, say that instead.)